.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+dist: trusty
```

Comment:
Nitpick: `trusty` support ended [2 months ago (01 Apr 2019)](https://wiki.ubuntu.com/Releases). It might make sense to use `xenial` (16.04) instead.

Reply:
Happy to accept a PR :)